website/layout/DocsLayout.js

```diff
@@ -17,6 +17,7 @@ var DocsLayout = React.createClass({
           <DocsSidebar metadata={metadata} />
           <div className="inner-content">
             <a id="content" />
+            <a className="edit-page-link" href={'https://github.com/facebook/jest/blob/master/docs/' + metadata.source} target="_blank">Edit on GitHub</a>
```

Comment:
I copied it from the Relay website, but did notice that all our websites do something different. Just tried to be consistent :)